programs/mango-v4/src/instructions/ix_gate_set.rs

```diff
@@ -0,0 +1,20 @@
+use anchor_lang::prelude::*;
+
+use crate::state::*;
+
+#[derive(Accounts)]
+pub struct IxGateSet<'info> {
+    #[account(
+        mut,
+        has_one = admin,
```

Comment:
should be available to the security admin (that is, disabling instructions should be available, not re-enabling)